src/Services/IMAPAuth.php

```diff
@@ -62,7 +62,12 @@ protected function imapOpen($username, $password)
                 $this->utils->createPasswordlessUserWithDefaultObjects($username, $username, $username);
 
                 $em = $this->doctrine->getManager();
-                $em->flush();
+
+                try {
+                    $em->flush();
+                } catch (\Exception $e) {
+                    error_log('IMAP Error (flush): '.$e->getMessage());
+                }
```

Comment:
Catching the exception will allow to continue (as the auth succeeded) and not log the trace, if any